handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -121,6 +124,29 @@ public SelectedListenerFailureBehavior selectedListenerFailureBehavior() {
         if (logger.isDebugEnabled()) {
             logger.debug("Default cipher suite (OpenSSL): " + ciphers);
         }
+
+        Integer dhLen = null;
+
+        try {
+            String dhKeySize = AccessController.doPrivileged(new PrivilegedAction<String>() {
+                @Override
+                public String run() {
+                    return SystemPropertyUtil.get("jdk.tls.ephemeralDHKeySize");
+                }
+            });
+            if (dhKeySize != null) {
+                try {
+                    dhLen = Integer.parseInt(dhKeySize);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method io.netty.handler.ssl.OpenSslContext.<static initializer for OpenSslContext>() converts String to boxed primitive using excessive boxing [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=fb-contrib%3ANAB_NEEDLESS_BOXING_VALUEOF)